FindTown/FindTown/Presentation/Common/Flow/TabBarCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TabBarCoordinator.swift
+//  FindTown
+//
+//  Created by 이호영 on 2022/12/25.
+//
+
+import UIKit
+import FindTownCore
+
+final class TabBarCoordinator: FlowCoordinator {
+    
+    var presentationStyle: PresentationStyle
+    weak var navigationController: UINavigationController?
+    
+    
+    init(presentationStyle: PresentationStyle) {
+        self.presentationStyle = presentationStyle
+    }
+    
+    func initScene() -> UIViewController {
+        let tabBarController: UITabBarController = BaseTabBarController()
+        
+        let homeCoordinator = HomeCoordinator(presentationStyle: .none)
+        homeCoordinator.start()
+        guard let homeViewController = homeCoordinator.navigationController else { return UIViewController() }
+        homeViewController.tabBarItem = UITabBarItem(title: "동네 찾기", image: UIImage(systemName: "house"), tag: 0)
+        
+        let mapCoordinator = MapCoordinator(presentationStyle: .none)
+        mapCoordinator.start()
+        guard let mapViewController = mapCoordinator.navigationController else { return UIViewController() }
+        mapViewController.tabBarItem = UITabBarItem(title: "동네 지도", image: UIImage(systemName: "map"), tag: 1)
+        
+        tabBarController.viewControllers = [homeViewController, mapViewController]
+        tabBarController.hidesBottomBarWhenPushed = true
+        return tabBarController
```

Comment:
```suggestion
        let tabBarController: UITabBarController = BaseTabBarController()
        
        let homeCoordinator = HomeCoordinator(presentationStyle: .none)
        let mapCoordinator = MapCoordinator(presentationStyle: .none)
        
        homeCoordinator.start()
        mapCoordinator.start()
       
        guard let homeViewController = homeCoordinator.navigationController else { return UIViewController() }
        guard let mapViewController = mapCoordinator.navigationController else { return UIViewController() }
        
```
이 부분 기존 작성해주신 코드에 주석을 추가하시거나 혹은 위처럼 유사한 코드끼리 텝바 순서대로 작성하는건 어떨까용?☺️